cmd/arcstat.in

```diff
@@ -29,7 +29,7 @@
 # with the License.
 #
 # You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
```

Comment:
If we're fixing up references we should probably update `usr/src/OPENSOLARIS.LICENSE` as well.  The top-level `LICENSE` file contains the mentioned copy of the license in the OpenZFS source tree.  This file in mentioned again in the next paragraph.

Reply:
I thought about that, but it looked like a lot more work... so I left this ;-)

But of cause, I could search for `CDDL HEADER START` and `CDDL HEADER END` ... and replace everything between them.

How should the new text look like?

```
/*
 * SPDX-License-Identifier: CDDL-1.0
 *
 * CDDL HEADER START
 *
 * The contents of this file are subject to the terms of the Common
 * Development and Distribution License (CDDL-1.0).
 *
 * You may not use this file except in compliance with the License.
 * You can obtain a copy of the license within the root directory of
 * OpenZFS or this link: https://opensource.org/licenses/CDDL-1.0
 * See the License for the specific language governing permissions
 * and limitations under the License.
 *
 * CDDL HEADER END
 */

Sample copyright's below the CDDL:

/*
 * Based on some work by XYZ released under CC0 or so
 * Copyright (c) 2022 Some Other <email@xyz>
 */

/*
 * Copyright (c) 2009 Sun Microsystems, Inc. All rights reserved.
 * Copyright (c) 2015 Nexenta Systems, Inc. All rights reserved.
 * Copyright (c) 2015 by Delphix. All rights reserved.
 */
```

Reply:
That really would make this a much more disruptive change.  Yes, let's just update the link as you've done which shouldn't be controversial